eng/NoTargetsSdk.BeforeTargets.targets

```diff
@@ -5,4 +5,4 @@
     <TargetFramework Condition="'$(TargetFramework)' == '' and '$(TargetFrameworks)' == ''">$(NetCoreAppCurrent)</TargetFramework>
   </PropertyGroup>
 
-</Project>
\ No newline at end of file
+</Project>
```

Comment:
Please just ignore this change. I originally had a change in that file and then reverted it in the GH UI. Don't want to spin another build for this...